src/ray/gcs/gcs_client/global_state_accessor.cc

```diff
@@ -129,6 +129,27 @@ std::unique_ptr<std::string> GlobalStateAccessor::GetObjectInfo(
   return object_info;
 }
 
+std::string GlobalStateAccessor::GetNodeResourceInfo(const ClientID &node_id) {
```

Comment:
If getting node resource info fails, how about return nullptr?

Reply:
It's empty map not nullptr, so we also pack them into protobuf streaming.

> If getting node resource info fails, how about return nullptr?

Reply:
Actually, All queries should return status if we want throw accessor failure.